modules/users/client/config/users.client.routes.js

```diff
@@ -101,20 +101,6 @@
         url: '/reset',
         template: '<ui-view/>'
       })
-      .state('password.reset.invalid', {
```

Comment:
The removal of this route is causing a 400 response on the server, with [this redirect](https://github.com/meanjs/mean/blob/master/modules/users/server/controllers/users/users.password.server.controller.js#L115). 

This keeps the User on the Reset Password view if they click on an invalid reset link (e.g. the token is expired). We should keep this route since the User should be taken away from the Reset Password view, in the case of an invalid token; they wouldn't be able to do anything there anyways.

I see there are some good reasons to get rid of the *Success* route, but by keeping it we provide more flexibility for our users. And we could always redirect somewhere else in the case of an *Invalid* token, but I don't see a need to change this flow now.

IMO, we should keep the flow with these two routes. There can probably be some improvements to the Reset Password feature, but I think it's a bit out of scope for this PR.